src/components/Bag/classes.js

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import { v4 as uuidv4 } from "uuid";
+import { Label, InsertDriveFile, Folder } from "@material-ui/icons";
+
+class BagItem {
+    constructor(item) {
+        this.item = item;
+    }
+
+    icon(t) {
+        return <Label />;
+    }
+
+    get label() {
+        return (
+            this.item?.label ||
+            this.item?.name ||
+            this.item?.description ||
+            "override label() please"
```

Comment:
Is this message for debugging or will this actually display to the user?

Reply:
I removed it and replaced it with an empty string.